jenkinsfile/aws/Jenkinsfile

```diff
@@ -53,12 +54,11 @@ node(NODE_MASTER) {
     //Config Files
     def ENV = '/home/jenkins/jenkins_env'
     def PUPPET_CONF = "${WORKSPACE}/${PIPELINE_RESOURCES_DIR}/scripts/puppet/apply-config.sh"
-    def DEV_CF = "${WORKSPACE}/${DEPLOYMENT_DIR}/sampleCFN/sample.yaml"
+    def ARTIFACT_CONF = "${WORKSPACE}/${ARTIFACTS_DIR}/buildArtifact.sh"
     def PACKER_MANIFEST = "${WORKSPACE}/${PIPELINE_RESOURCES_DIR}/scripts/packer/manifest.json"
     def PACKER_JSON = "${WORKSPACE}/${PIPELINE_RESOURCES_DIR}/scripts/packer/packer.json"
-    def AWS_CREDS_LOC = '/home/ubuntu/.aws/credentials'
     //repos
-    def GIT_REPO_RESOURCES = 'https://github.com/wso2/aws-pipeline'
+    def GIT_REPO_RESOURCES = 'https://github.com/shafanaS/aws-pipeline'
```

Comment:
@shafanaS shouldn't this change be reverted since it points to your private fork?

Reply:
Done